teku/src/main/java/tech/pegasys/teku/cli/subcommand/debug/PrettyPrintCommand.java

```diff
@@ -59,24 +59,24 @@ public class PrettyPrintCommand implements Callable<Integer> {
       paramLabel = "<FORMAT>",
       description = "Output format to use",
       arity = "1")
-  @SuppressWarnings("FieldMayBeFinal")
+  @SuppressWarnings("FieldCanBeFinal")
   private OutputFormat format = OutputFormat.JSON;
 
   @Option(
       names = {"-n", "--network"},
       paramLabel = "<NETWORK>",
       description = "Network config to use when decoding objects.",
       arity = "1")
-  @SuppressWarnings("FieldMayBeFinal")
+  @SuppressWarnings("FieldCanBeFinal")
   private String network = "mainnet";
 
   @Parameters(
       index = "0",
       paramLabel = "<MILESTONE>",
       description = "Milestone to use when decoding objects.",
       arity = "1")
-  @SuppressWarnings("FieldMayBeFinal")
-  private SpecMilestone milestone = SpecMilestone.ALTAIR;
+  @SuppressWarnings("FieldCanBeFinal")
+  private final SpecMilestone milestone = SpecMilestone.ALTAIR;
```

Comment:
you should remove `final` here?

Reply:
Think the patch was applied here because of the outdated annotation.
I probably forgot to remove this when I fixed the annotation. Thanks for catching this!